yang/ietf-packet-discard-reporting-sx.yang

```diff
@@ -366,7 +366,9 @@ module ietf-packet-discard-reporting-sx {
       type yang:counter64;
       description
         "The number of received packets discarded due to an
-         invalid Segment Routing (SR) SID.";
+         invalid Segment Routing over IPv6 (SRv6) SID.
+         For SR-MPLS, invalid SIDs encoded as MPLS labels
```

Comment:
This may be interpreted as "only those SIDs encoded as MPLS are considered", while SIDs are always encoded as label for SR-MPLS. I would delete the ""encoded as MPLS lablels"